target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom

```diff
@@ -3,6 +3,7 @@
 [ -e /lib/firmware/$FIRMWARE ] && exit 0
 
 . /lib/functions/caldata.sh
+. /lib/functions/mikrotik.sh
```

Comment:
Call this caldata-mikrotik.sh or mikrotic-caldata.sh.

Reply:
:+1: